src/components/VaccinationEligibilityBlock/VaccinationEligibilityBlock.tsx

```diff
@@ -76,7 +76,7 @@ const VaccinationEligibilityBlock: React.FC<{ region: Region }> = ({
 
   return (
     <Container>
-      <Heading2>Vaccine eligibility</Heading2>
+      <Heading2 style={{ marginTop: 0 }}>Vaccine eligibility</Heading2>
```

Comment:
putting this style inline for now. going to unify all of the section headings in a separate pr